packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.ts

```diff
@@ -213,4 +219,16 @@ export class IframeEnclave implements EnclaveProvider {
       console.error(error);
     }
   }
+
+  async discoverUserEncryptionKey(humanId: string): Promise<DiscoverEncryptionKeyResponse> {
+    if (this.options.mode !== "new")
+      throw new Error("You can only call discoverUserEncryptionKey when mode is 'new'.");
+
+    const encryptionPublicKey = await this.ready(humanId);
+
+    return {
+      humanId,
+      encryptionPublicKey: Base64Codec.encode(encryptionPublicKey as Uint8Array),
```

Comment:
Why are we forcing a type here? My editor is ok without it.

```suggestion
      encryptionPublicKey: Base64Codec.encode(encryptionPublicKey),
```